plugins/GSdx/GSRendererDX11.cpp

```diff
@@ -37,10 +43,54 @@ bool GSRendererDX11::CreateDevice(GSDevice* dev)
 	return true;
 }
 
-void GSRendererDX11::SetupIA()
+void GSRendererDX11::SetupIA(const float& sx, const float& sy)
 {
 	GSDevice11* dev = (GSDevice11*)m_dev;
 
+	D3D11_PRIMITIVE_TOPOLOGY t;
+
+	bool unscale_hack = UserHacks_unscale_pt_ln  & (GetUpscaleMultiplier() > 1);// &GLLoader::found_geometry_shader;
```

Comment:
> (GetUpscaleMultiplier() > 1)

Won't work for custom resolutions in that case, intended?

Also FWIW, it would be nice to limit the usage of bitwise AND ``(&)`` only when comparing bitfields or for bitwise manipulations, logical AND ``(&&)`` makes more sense here.